disnake/threads.py

```diff
@@ -193,6 +206,7 @@ def _from_data(self, data: ThreadPayload):
         self.last_message_id = _get_as_snowflake(data, "last_message_id")
         self.slowmode_delay = data.get("rate_limit_per_user", 0)
         self.message_count = data.get("message_count")
+        self.total_message_sent = data.get("total_message_sent")
```

Comment:
in theory, both of these could result in message_count and total_message_sent being `None`.

Reply:
> in theory, both of these could result in message_count and total_message_sent being `None`.

@onerandomusername so should we suppose if they're `None` they're 0?